server/config/router.js

```diff
@@ -33,22 +35,22 @@ module.exports = function (nextjs, koa, config) {
     Authenticated routes
   \******************************************************************************/
 
-  let authenticatedRoutes = [
-    '/admin/*',
-    '/paperwork',
-    '/paperwork/*',
-    '/profile',
-    '/authorize'
-  ]
-
-  router.get(authenticatedRoutes, async (ctx, next) => {
-    if (ctx.cookie && ctx.cookie.access_token) {
-      await next()
+  // let authenticatedRoutes = [
+  //   '/admin/*',
+  //   '/paperwork',
+  //   '/paperwork/*',
+  //   '/profile',
+  //   '/authorize'
+  // ]
 
-    } else {
-      await ctx.redirect(`/?authenticate=true&destination=${encodeURIComponent(ctx.request.url)}`)
-    }
-  })
+  // router.get(authenticatedRoutes, async (ctx, next) => {
+  //   if (ctx.cookie && ctx.cookie.access_token) {
+  //     await next()
+  //
+  //   } else {
+  //     await ctx.redirect(`/?authenticate=true&destination=${encodeURIComponent(ctx.request.url)}`)
+  //   }
+  // })
```

Comment:
Where did the code for the authenticated routes go? Is this handled by the new router?

Reply:
Authenticated routes are defined and handled by the page itself instead of the router now.

See [this](https://github.com/FuelRats/fuelrats.com/pull/50/files/3715894eb2e42aafc508f3026bc6f66d5496df22#diff-2465a0749a2041e522b6b8d7681adf5aR62)